app/views/search/_result.html.erb

```diff
@@ -74,7 +74,7 @@
   
   <% if result.url %>
     <div class="result-get">
-      <%= link_to('Get it', result.get_it_url, class: 'button button-secondary') %>
+      <%= link_to('Get it', result.get_it_url, class: 'button button-primary') %>
```

Comment:
FYI: we're already setting the `result.get_it_url` in the normalizer classes. We can easily create and set a `result.get_it_label` to give us more control over the labeling.